src/helperFunctions/findCoordinates.ts

```diff
@@ -0,0 +1,20 @@
+export default function findCoordinates() {
+    const searchBar = document.querySelector('#search-bar-div');
+    const filterContainer = document.querySelector('#filter-container');
+    const input = document.querySelector('input');
```

Comment:
input querySelector can be more specific in case there are more input elements.
For example, tasks edit mode for super users have other input elements in this page.

Reply:
It is always going to be the first input element in the entire HTML.

Reply:
Anyway I've updated it with className.